DashAI/back/explainability/global_explainer.py

```diff
@@ -0,0 +1,16 @@
+from abc import abstractmethod
+
+from DashAI.back.dataloaders.classes.dashai_dataset import DashAIDataset
+from DashAI.back.explainability.base_explainer import BaseExplainer
+
+
+class GlobalExplainer(BaseExplainer):
```

Comment:
Maybe a small docstring commenting on what is a global explainer and the explain method?